spec/models/piece_spec.rb

```diff
@@ -225,44 +225,39 @@
       expect(king.y_position).to eq 3
     end
 
-    # it 'should move the piece to the destination square; if the square is occupied, mark that piece as captured' do
-    #   game = Game.create(white_user_id: 0, black_user_id: 1)
-    #   king = King.create(x_position: 4, y_position: 2, game_id: game.id)
-    #   rook = Rook.create(x_position: 4, y_position: 3, game_id: game.id)
-    #   king.move_to!(4,3)
-    #   expect(king.x_position).to eq 4
-    #   expect(king.y_position).to eq 3
-    #   expect(rook.captured).to eq true
-    # end
-  end
-
-  describe 'white?' do
-    it 'should return true for white piece' do
-      game = Game.create(white_user_id: 0, black_user_id: 1)
-      pawn = Pawn.create(x_position: 0, y_position: 6, game_id: game.id, user_id: 0)
-      expect(pawn.white?).to be true
+    # test is failing - true is expected, returns nil (the default value)
+    it 'should move the piece to the destination square; if the square is occupied, mark that piece as captured' do
+      white_user = FactoryGirl.create(:user)
+      black_user = FactoryGirl.create(:user)
+      game = Game.create(white_user_id: white_user.id, black_user_id: black_user.id)
+      king = King.create(x_position: 4, y_position: 2, game_id: game.id, user_id: white_user.id)
+      rook = Rook.create(x_position: 4, y_position: 3, game_id: game.id, user_id: black_user.id)
+      king.move_to!(4,3)
+      expect(king.x_position).to eq 4
+      expect(king.y_position).to eq 3
+      expect(rook.captured).to eq true
     end
-  end
 
-  describe 'black?' do
-    it 'should return true for black piece' do
-      game = Game.create(white_user_id: 0, black_user_id: 1)
-      pawn = Pawn.create(x_position: 0, y_position: 1, game_id: game.id, user_id: 1)
-      expect(pawn.black?).to be true
+    # I thought perhaps the test was returning nil because the rook was marked captured, and therefore inactive, aka out of scope, so the entire value was nil. That's not true.
+    it 'should move the piece to the destination square; the captured piece will return as nil' do
+      white_user = FactoryGirl.create(:user)
+      black_user = FactoryGirl.create(:user)
+      game = Game.create(white_user_id: white_user.id, black_user_id: black_user.id)
+      king = King.create(x_position: 4, y_position: 2, game_id: game.id, user_id: white_user.id)
+      rook = Rook.create(x_position: 4, y_position: 3, game_id: game.id, user_id: black_user.id)
+      king.move_to!(4,3)
+      expect(king.x_position).to eq 4
+      expect(king.y_position).to eq 3
+      expect(rook).to eq nil # returns #<Rook id: 13335, type: "Rook", user_id: 98, game_id: 6434, created_at: "2016-08-03 22:56:10", updated_at: "2016-08-03 22:56:10", x_position: 4, y_position: 3, captured: nil>
```

Comment:
This won't ever equal nil - it's a reference to the rook object, and no matter what you do to it internally you won't be able to update an outside reference.  Not sure exactly what you want to test here (most likely that captured is true).